source/github-helpers/pr-ci-status.ts

```diff
@@ -6,7 +6,8 @@ type CommitStatus = false | typeof SUCCESS | typeof FAILURE | typeof PENDING | t
 type StatusListener = (status: CommitStatus) => void;
 
 // `.TimelineItem--condensed` excludes unrelated references. See `deemphasize-unrelated-commit-references` feature
-const commitSelector = '.js-commit-group .TimelineItem--condensed';
+// TODO [2022-05-01]: Remove `.js-commit-group` (GHE)
+const commitSelector = ':is(.js-commit-group, [data-test-selector="pr-timeline-commits-list"]) .TimelineItem--condensed';
```

Comment:
Seems like the `js-commit-group` class doesn't exist anymore.

The only other feature making use of it is `parse-backticks`:
https://github.com/refined-github/refined-github/blob/702407e6b9a5cbfe0e82ef6aa6f634bdcc762b2f/source/features/parse-backticks.tsx#L12-L13

The first line doesn't seem needed anymore:

![image](https://user-images.githubusercontent.com/46634000/142772634-dc06c01a-7d39-476f-9904-2918f0d25719.png)

I don't know about the second.